pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ClientCachedDao.java

```diff
@@ -0,0 +1,119 @@
+package org.broadinstitute.ddp.db.dao;
```

Comment:
Our first implementation in the PR of a wrapped cached DAO (Is there as way I can change the order these files are listed?).
Is a good example!